src/app/components/main-page-buttons/main-page-buttons.component.html

```diff
@@ -0,0 +1,6 @@
+<div id="button-bar" *ngIf="userAuthenticated" style="margin-bottom: 1rem;">
+    <button type="button" (click)="startNewIncident()">
+        <fa-icon [icon]="icons.newElement"></fa-icon>
+        <span>New Incident</span>
+    </button>
+</div>
```

Comment:
```suggestion
</div>

```

Nitpick: Missing new line.